### PR TITLE
Further snippet clean up

### PR DESCRIPTION
**Pull request by AlexJerabek**
- Fixing Chart Title HTML 
- Use console.log for conditional format display 
- Fixing spacing 
- Clean up PivotTable sample HTML formatting

### CODE REVIEW

File: samples/excel/14-conditional-formatting/conditional-formatting-basic.yaml [hunk 1]
```diff
@@ -171,19 +171,12 @@ script:
...
-                            `${item.cf.type}`)
-                        let $li = $(`<li></li>`);
-                        $li.append($p);
-                        $("#conditional-formats").append($li);
-                        $(".conditional-formats").show()[0].scrollIntoView();
-                    })
-                }
-                else {
-                    OfficeHelpers.UI.notify("None to display", "No conditional formats in workbook", "warning");
+                cfRangePairs.forEach(item => {
```
Comment (kbrandl) [inline samples/excel/14-conditional-formatting/conditional-formatting-basic.yaml]: lines within the `if` should be indented

File: samples/excel/14-conditional-formatting/conditional-formatting-basic.yaml [hunk 2]
```diff
@@ -171,19 +171,12 @@ script:
...
+                    console.log(item.cf.type);
+                });
+                } else {
+                console.log("No conditional formats applied.");
```
Comment (kbrandl) [inline samples/excel/14-conditional-formatting/conditional-formatting-basic.yaml]: line within the `else` should be indented

File: snippet-extractor-output/snippets.yaml [hunk 3]
```diff
@@ -1312,19 +1310,12 @@ Excel.ConditionalFormat.type:
...
-                $(".conditional-formats").show()[0].scrollIntoView();
-            })
-        }
-        else {
-            OfficeHelpers.UI.notify("None to display", "No conditional formats in workbook", "warning");
+        cfRangePairs.forEach(item => {
+            console.log(item.cf.type);
+        });
+        } else {
+        console.log("No conditional formats applied.");
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: `console.log` line needs to be indented

File: snippet-extractor-output/snippets.yaml [hunk 4]
```diff
@@ -1346,19 +1337,12 @@ Excel.ConditionalFormat.getRange:
...
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: `console.log` line needs to be indented

File: snippet-extractor-output/snippets.yaml [hunk 5]
```diff
@@ -1346,19 +1337,12 @@ Excel.ConditionalFormat.getRange:
...
-                    `${item.cf.type}`)
-                let $li = $(`<li></li>`);
-                $li.append($p);
-                $("#conditional-formats").append($li);
-                $(".conditional-formats").show()[0].scrollIntoView();
-            })
-        }
-        else {
-            OfficeHelpers.UI.notify("None to display", "No conditional formats in workbook", "warning");
+        cfRangePairs.forEach(item => {
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: lines within `if` need to be indented

File: snippet-extractor-output/snippets.yaml [hunk 6]
```diff
@@ -1312,19 +1310,12 @@ Excel.ConditionalFormat.type:
...
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: lines within `if` need to be indented

File: snippet-extractor-output/snippets.yaml [hunk 7]
```diff
@@ -1396,19 +1380,12 @@ Excel.ConditionalFormatCollection.getItemAt:
...
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: lines within `if` need to be indented

File: snippet-extractor-output/snippets.yaml [hunk 8]
```diff
@@ -1396,19 +1380,12 @@ Excel.ConditionalFormatCollection.getItemAt:
...
+            console.log(item.cf.type);
+        });
+        } else {
```
Comment (kbrandl) [inline snippet-extractor-output/snippets.yaml]: line within `else` needs to be indented

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT  and @kbrandl. Indentation updated.

**Pull request merged**